blog/2024-09-10-pants-2-22/index.mdx

```diff
@@ -0,0 +1,62 @@
+    Unsplash
+  </a>
+</CaptionedImg>
+
+{/* truncate */}
+
+We are pleased to announce Pants [2.22.0](<(https://github.com/pantsbuild/pants/tree/2.22.x/src/python/pants/notes/2.22.x.md)>), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.22.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/docs/getting-started).
+
+_Highlights in [2.22](<(https://github.com/pantsbuild/pants/tree/2.22.x/src/python/pants/notes/2.22.x.md)>) include_:
+
+- 🎛️A major re-implementation of the Pants [options system](https://www.pantsbuild.org/2.22/docs/using-pants/key-concepts/options). See below for more!
+- 🏠Support for executing local processes within the repository itself (rather than in a sandbox) via the new [workspace environment](https://www.pantsbuild.org/stable/docs/using-pants/environments#in-workspace-execution-experimental_workspace_environment) feature.
+- 🐍Initial support for Python 3.13.
+- 🛫Ability to export Python tools [from their default bundled lockfiles](https://www.pantsbuild.org/2.22/docs/using-pants/setting-up-an-ide#tool-virtualenvs).
+- ☁️Improvements in FaaS support, such as the ability to create cloud functions as directories, instead of .zip files.
+- 🏷️[`vcs_version`](https://www.pantsbuild.org/stable/reference/targets/vcs_version) always executes locally, so that it has the local git context, even when otherwise using docker environments.
+- 📊A new experimental SQL backend, with [SQLFluff](https://sqlfluff.com/) linter support.
+- 🗃️A new experimental [Makeself](https://makeself.io/) backend for creating self-extractable archives.
+- 🫙A [`deploy_jar_exclude_files`](https://www.pantsbuild.org/2.22/reference/subsystems/jvm#deploy_jar_exclude_files) option, to allow excluding irrelevant signature files from JAR files.
+- 🗺️Better understanding of self-type annotations in Scala dependency inference.
+- 🎒the ability to bundle `openapi_document` and its `openapi_source` dependencies into a single file, for consumption by other targets.
+- 📈 And lots of smaller features, update, bugfixes, and general improvements
+
+Check out the [full release notes](https://github.com/pantsbuild/pants/tree/2.22.x/src/python/pants/notes/2.22.x.md). Pants is an open-source project, and the changes are all contributed by our community. If you want to see something more in the next changelog, join us on [GitHub](https://github.com/pantsbuild/pants) and become a contributor.
+
+In particular, please note the transition to the new options system. In 2.22.x we run both the new and the legacy systems concurrently and compare the results, issuing warnings if there are any discrepancies. If you do encounter discrepancies that you can't resolve on your own, please [reach out to us!](https://www.pantsbuild.org/docs/getting-help).
```

Comment:
I think the canonical url is now https://www.pantsbuild.org/community/getting-help